PayPal/src/main/java/com/braintreepayments/api/PayPalVaultRequest.java

```diff
@@ -47,6 +63,10 @@ String createRequestBody(Configuration configuration, Authorization authorizatio
             parameters.put(DESCRIPTION_KEY, billingAgreementDescription);
         }
 
+        if (userAuthenticationEmail != null) {
```

Comment:
General non-blocking question, does the API throw any sort of error if we send an empty string in here or is java smart enough to equate an empty string to null?

Reply:
I'd have to double-check, but my guess is that an empty string would be set on this JSON property. Do you know if we're checking for an empty string on iOS?

Reply:
🤔 Just looked through the PR on iOS and we are only using `if let` which does the same thing you have (just checks that it's not nil). I guess maybe the question is _should_ we be checking that it's not just an empty string?

Reply:
i don't feel strongly one way or the other. it wouldn't hurt to add the empty check, also the API probably ignores an empty email address as well.

maybe we leave it as is and pipe the empty string to the API? less clientside logic to maintain 🤔

Reply:
That makes sense, as long as the API doesn't throw an error if we pass in an empty string I am a-ok with keeping it as is

Reply:
Just tested the API sending in an empty string. No error is thrown!

Reply:
Perfect! I'll resolve this.